pkgs/by-name/as/asset-ripper/package.nix

```
@@ -0,0 +1,65 @@
+{
+  autoPatchelfHook,
+  buildDotnetModule,
+  dbus,
+  dotnetCorePackages,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildDotnetModule (finalAttrs: {
+  pname = "AssetRipper";
+  version = "1.1.3";
+
+  src = fetchFromGitHub {
+    owner = "AssetRipper";
+    repo = "AssetRipper";
+    rev = "refs/tags/${finalAttrs.version}";
```

Please rebase on master branch and do this:
```suggestion
    tag = finalAttrs.version;
```
See #355973.

---

I've tried rebasing locally and it doesn't build. I think something the fault is in Nixpkgs, but I haven't finished bisecting.

---

Could you share the build error you are receiving? If it is related to missing lib deps in two of the nuget precompiled libraries, then I have overcome that by including `stdenv.cc.cc.lib` in the `buildInputs` as a temporary fix. Would be interested in finding the breaking commit.

---

That was it, yeah.
```
searching for dependencies of /nix/store/y5y38bj32dd71ki3q1k0slnm2jals223-AssetRipper-1.1.3/lib/AssetRipper/libTexture2DDecoderNative.so
    libstdc++.so.6 -> not found!
    libgcc_s.so.1 -> not found!
auto-patchelf: 2 dependencies could not be satisfied
error: auto-patchelf could not satisfy dependency libstdc++.so.6 wanted by /nix/store/y5y38bj32dd71ki3q1k0slnm2jals223-AssetRipper-1.1.3/lib/AssetRipper/libTexture2DDecoderNative.so
error: auto-patchelf could not satisfy dependency libgcc_s.so.1 wanted by /nix/store/y5y38bj32dd71ki3q1k0slnm2jals223-AssetRipper-1.1.3/lib/AssetRipper/libTexture2DDecoderNative.so
auto-patchelf failed to find all the required dependencies.
Add the missing dependencies to --libs or use `--ignore-missing="foo.so.1 bar.so etc.so"`.
```
Culprit being 687371ab28358ca878a93f4ef4884bfecf0ac40c (not sure how removing Python from propagated deps means removing `libstdc++` but there you go).